src/components/Container.tsx

```diff
@@ -26,18 +34,27 @@ export default function Container({ containerModel }: CotaninerProps) {
       }}
     >
       <Tabs value={tabValue} onChange={handleChange}>
-        {layout.children?.map((chart: TODO) => (
+        {children.map((chart: TODO) => (
           <Tab
             id={`container-tab-${chart.refId}`}
```

Comment:
We need a `data-testid: ... ` for all interactive components so they can be found up when we write tests.

Reply:
data-testid added